kubeclient.gemspec

```diff
@@ -30,4 +30,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rest-client'
   spec.add_dependency 'recursive-open-struct', '~> 1.0.4'
   spec.add_dependency 'http', '~> 2.2.2'
+  spec.add_dependency 'json-streamer'
```

Comment:
I'm unfamiliar so can't recommend anything specific. But I think that it should be taken into consideration shall fixes/bugs will be needed in that lib, and there's only one contributor (so there's a risk that he/she also stop developing it in any moment)